pytest_client_tools/plugin.py

```diff
@@ -91,15 +92,20 @@ def candlepin(request):
             "--candlepin-container-is-running"
         )
         initial_wait = 0
+        verify = False
         if start_container:
-            stack.enter_context(_create_candlepin_container())
+            container = stack.enter_context(_create_candlepin_container())
             # candlepin takes a while to start
             initial_wait = 5
+            temp_cert = stack.enter_context(tempfile.NamedTemporaryFile())
+            container.cp("$C:/etc/candlepin/certs/candlepin-ca.crt", temp_cert.name)
+            verify = temp_cert.name
```

Comment:
I noticed now that pytest has something nice to manage temporary directories for session-scoped fixtures: the `tmp_path_factory` fixture; you can see it described here:
https://docs.pytest.org/en/8.2.x/how-to/tmp_path.html#tmp-path-factory-example
Using that, we don't need to create an own temporary file, and everything is within the global temporary directory of the pytest session; I just tested this, and it seems to work:
```python
            ca_cert = str(tmp_path_factory.mktemp("candlepin") / "ca.crt")
            container.cp("$C:/etc/candlepin/certs/candlepin-ca.crt", ca_cert)
            verify = ca_cert
```
would you please give it a try, and in case it seems worth amend this?